WebApplication/Worker.cs

```diff
@@ -53,7 +53,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                 }
                 catch(Exception e)
                 {
-                    _logger.LogError(e.Message);
+                    _logger.LogError($"Fatal error during migration process !!!\n{e.Message}\n{e.StackTrace}");
```

Comment:
BTW - exception can be logged with special overload method: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.logerror?view=dotnet-plat-ext-3.1#Microsoft_Extensions_Logging_LoggerExtensions_LogError_Microsoft_Extensions_Logging_ILogger_System_Exception_System_String_System_Object___

Reply:
Thanks I will fix it in subsequent PR